packages/fiori/src/DynamicPage.ts

```diff
@@ -226,6 +226,7 @@ class DynamicPage extends UI5Element {
 			this.dynamicPageTitle.snapped = this._headerSnapped;
 			this.dynamicPageTitle.interactive = this.hasHeading;
 			this.dynamicPageTitle.hasSnappedTitleOnMobile = !!this.hasSnappedTitleOnMobile;
+			this.dynamicPageTitle?.removeAttribute("hovered");
```

Comment:
Minor, could you please remove the optional chaining operator `?`, because  the `if` statement above ensures that `this.dynamicPageTitle` is not `null` or `undefined`.